buildscripts/BuildScripts.csproj

```diff
@@ -1,18 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net461;netcoreapp1.1</TargetFrameworks>
+		<TargetFramework>net5.0</TargetFramework>
```

Comment:
@jonorossi is this a good change, I don't see the need for the old stuff. Gavin added to reflect "look and feel" of castle.core.

Reply:
It would have likely also been to support Mono at the time.

I assume this sets the default for packages, seems like a decent default, however would we be better off with no default?

Reply:
I think Gavin just copied from caste.core. The project is only a wrapper for some scripts, there is nothing to build in the project, if I unload, rebuild is still green.

Should I remove in favour of a folder?

Reply:
It might have been to keep the files in `BuildScripts.csproj` in the solution.

Reply:
We can just use a solution folder. with a nice name "BuidlScripts".. Shoud i change?

Reply:
Sure, go for it as long as that's "BuildScripts" 😉